design/lifecycle.md

```diff
@@ -0,0 +1,15 @@
+# Container Lifecycle
+
+While containerd is a daemon that provides API to manage multiple containers the containers themselves are not tied to the lifecycle of containerd.  Each container has a shim that acts as the direct parent for the container's processes as well has reporting the exit status and holding onto the STDIO of the container.  This also allows containerd to crash and restore all functionality to containers.  
+
+
+## containerd
+
+The daemon provides an API to manage multiple containers.  It can handle locking in process where needed to cordinate tasks between subsystems.  While the daemon does fork off the needed processes to run containers, the shim and runc, these are re-parented to the system's init.
```

Comment:
it might be worthwhile to intentionally double-fork to force the adoption, but it's a relatively minor thing,